lib/matplotlib/markers.py

```diff
@@ -303,6 +315,13 @@ def get_path(self):
     def get_transform(self):
         return self._transform.frozen()
 
+    def set_transform(self, transform):
+        """
+        Sets the transform of the marker. This is the transform by which the
```

Comment:
First line of docstring should be separate.
```suggestion
        Sets the transform of the marker.
        
        This is the transform by which the marker path is transformed.
```